### PR TITLE
:rocket: Added type hints :rocket:

### PR DESCRIPTION
**Pull request by JoeyMcQuade**

### CODE REVIEW

File: get_stock_data.py [hunk 1]
```diff
@@ -8,37 +10,40 @@
...
-"""
-def get_ticker_symbol(company):
+def get_ticker_symbol(company: str) -> List[dict]:
+    """
+    Gets a list of suggestions for ticker symbols based on user input.
+    """
     # send a GET request with the company name as keyword in the url
     res = requests.get(f'https://www.alphavantage.co/query?function=SYMBOL_SEARCH&keywords={company}'
-                       f'&apikey={SECRET_API_KEY}')
+                       f'&apikey={config("SECRET_API_KEY")}')
```
Comment (rcampbell1337) [inline get_stock_data.py:19]: why did you change this

Comment (JoeyMcQuade) [reply get_stock_data.py:19]: This is more secure

**Review by rcampbell1337 — APPROVED** [review]
Very good
